File: 08320.py
rec = [0, 1]

def cnt(n):
    start = 1; end = n
    cnt = 0
    while True:
        if start > end:
            break

        if n % start == 0:
            cnt += 1
            end = n // start - 1
        start += 1

    return cnt

n = int(input())
for i in range(2, n + 1):
    rec.append(rec[i - 1] + cnt(i))

print(rec[n])

File: 08320.cpp
#include <iostream>
using namespace std;

int rec[10001];

int cnt(int n) {
	int start = 1, end = n;
	int cnt = 0;
	while (1) {
		if (start > end) break;

		if (n % start == 0) {
			cnt++;
			end = n / start - 1;
		}
		start++;
	}

	return cnt;
}

int main() {
	int n; cin >> n;

	rec[1] = 1;
	for (int i = 2; i <= n; i++)
		rec[i] = rec[i - 1] + cnt(i);

	cout << rec[n] << endl;
}
